### DISCUSSION TITLE
[Sequel::Migrator] ERROR:  relation \"schema_migrations\" does not exist

### DISCUSSION THREAD

**Discussion by rafryb** [jeremyevans/sequel#2018] [General]
Hello i have this error when i'm migrating my db:
<img width="1505" alt="Screenshot 2023-04-12 at 16 58 39" src="https://user-images.githubusercontent.com/8711270/231499746-a54f1c01-6e0d-4a2c-9f1b-e58000b5d9cd.png">

Migrations are passing correctly after and process is creating  "schema_migrations". My ci is considering this situation as a problem, I have currently "sql_log_level" set on "debug". What is best solution here which can be applied?

tested on version of sequel 5.67.0 and 5.58.0

**Comment by jeremyevans** [comment]
That's just Sequel checking whether the table exists.  If the table does not exist, Sequel creates it.  I'm not sure why your CI is considering this situation as a problem.

**Reply by rafryb** [reply]
@jeremyevans Thanks for fast reply, it seems then that i will need to apply some kind of filtering in github actions.

**Reply by AlexWayfer** [reply]
> i will need to apply some kind of filtering in github actions

What kind of filtering? Filtering of what?

It seems like I had similar issue but don't remember solution. Also I'd like to help if needed.

BTW, how do you run migrations? I use `Sequel::Migrator.run` and nothing more. Except database creation. The error seems you already have the DB. Do the migration connection has priviliges to create tables?

**Reply by rafryb** [reply]
@AlexWayfer I meant some custom log parser which would skip errors related to undefined tables.

I use currently rom rb which is relying on preinitialized Sequel client. Tested it also with pure Sequel::Migrator. 

Whats interesting problem disappeared today so most likely it was strictly related to github actions ci. After rerun it is not taking PG exception under consideration.

Thanks for the input.

**Reply by AlexWayfer** [reply]
> @AlexWayfer I meant some custom log parser which would skip errors related to undefined tables.

I'd not skip (ignore) such errors, they don't seem minor. But I'm glad this has been auto-resolved.
